### PR TITLE
[BIOIN-1209] Added option for coverage collection to only collect depth files

### PR DESCRIPTION
**Pull request by ilyasoifer**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -12,6 +12,7 @@
 *.bw filter=lfs diff=lfs merge=lfs -text
 *.interval_list filter=lfs diff=lfs merge=lfs -text
 *.bedGraph filter=lfs diff=lfs merge=lfs -text
+*.bedgraph filter=lfs diff=lfs merge=lfs -text
```
Comment (kenissur) [inline .gitattributes:15]: maybe this line was duplicates by mistake?

File: ugvc/pipelines/coverage_analysis.py [hunk 2]
```diff
@@ -241,6 +341,7 @@ def run_full_coverage_analysis(
         "-a",
         "-J",
         ref_str,
+        ref_str,
```
Comment (kenissur) [inline ugvc/pipelines/coverage_analysis.py]: duplicated by mistake again?

File: test/system/test_coverage_analysis.py [hunk 3]
```diff
@@ -31,3 +32,26 @@ def test_coverage_analysis(tmpdir):
     df = pd.read_hdf(pjoin(tmpdir, "170201-BC23.coverage_stats.q0.Q0.l0.h5"), "percentiles")
     df_ref = pd.read_parquet(f_ref)
     assert np.allclose(df.fillna(-1), df_ref.fillna(-1))
+
+
+def test_coverage_collection(tmpdir):
```
Comment (kenissur) [inline test/system/test_coverage_analysis.py:37]: I didn't know this tmpdir input option - cool!

**Review by doron-st — APPROVED** [review]
Can you add a few words on the difference between the full analysis and the limited analysis?
Not sure I got it

**Review by kenissur — APPROVED** [review]
Looks good, I should have separated it this way originally :)

Just comments about possible line duplications

File: .gitattributes (hunk 1 above)
Comment (ilyasoifer) [reply .gitattributes:15]: bed**g**raph versus bed**G**raph.

File: test/system/test_coverage_analysis.py (hunk 3 above)
Comment (ilyasoifer) [reply test/system/test_coverage_analysis.py:37]: :+1

File: ugvc/pipelines/coverage_analysis.py (hunk 2 above)
Comment (ilyasoifer) [reply ugvc/pipelines/coverage_analysis.py]: Fixed

**Pull request merged**